src/camvidlog2/yoloe/data.py

```
@@ -0,0 +1,38 @@
+from pathlib import Path
+from typing import Iterable
+
+import numpy as np
+import pandas as pd
+
+
+def create(
+    video: Path, trackings: Iterable[tuple[int, np.ndarray]], names: Iterable[str]
```

Ruff (0.11.9)

Unused function argument: `video`

(ARG001)